atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1426,12 +1427,15 @@ private void createTablesInternal(final Map<TableReference, byte[]> tableNamesTo
         clientPool.runWithRetry(client -> {
             for (Entry<TableReference, byte[]> tableEntry : tableNamesToTableMetadata.entrySet()) {
                 try {
+                    if (client.describe_schema_versions().containsKey("UNREACHABLE")) {
```

Comment:
This will be reverted for the moment, we believe that we can achieve this call against a degraded cluster.